config/initializers/sentry.rb

```diff
@@ -0,0 +1,48 @@
+require "sentry-raven"
+
+Raven.configure do |config|
+  config.dsn = ENV["SENTRY_DSN"]
+  config.environments = %w[staging production]
+  config.current_environment = Rails.env.to_s
+
+  if File.exist?(Rails.root.join("VERSION"))
+    config.release = File.read(Rails.root.join("VERSION")).strip
+  end
```

Comment:
Don't we have a `version` helper somewhere? It sounds odd to have all this logic here instead of doing `config.release = App.version` or something like that.